src/components/calcite-input-date-picker/calcite-input-date-picker.tsx

```diff
@@ -117,6 +119,21 @@ export class CalciteInputDatePicker {
     this.active = false;
   }
 
+  //--------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  //--------------------------------------------------------------------------
+  /**
+   * Trigger calcite date change when a user changes the date.
+   */
+  @Event() calciteDatePickerChange: EventEmitter<Date>;
```

Comment:
Isn't this creating a new unused event on the component though? 

Maybe we need to figure out a better way to document public events for components used internally?

What do you think @jcfranco?

Reply:
I'm not sure how you'd get an event into the component types without the decorator. I don't think it would add much overhead as it's never called, not totally sure what the stencil implementation is, though.

Reply:
I think this is the simplest way to have the build generate the doc for a delegated event. We do this for `calcite-value-list-item` too. It does generate an unused emitter in the component though.

Just brainstorming, but we could possibly use the `docs-custom` output target along with a custom JSDoc tag for delegated events. Then we would need to transform the custom docs JSON into markdown files.

Reply:
A custom jsdoc tag might be a cleaner solution long term but i'll leave it up to you guys.